examples/abitest/module2/rust/src/lib.rs

```diff
@@ -37,15 +37,15 @@ pub fn main(in_handle: u64) -> i32 {
     oak::set_panic_hook();
 
     // We expect a single empty message holding a reply channel.
-    let in_channel = oak::ReadHandle { handle: in_handle };
+    let in_channel = oak::ReadHandle { handle: oak::Handle::new(in_handle) };
```

Comment:
Also change the type of the `in_handle` arg to `oak::Handle`, and perform the conversion in `oak_main`?